dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
I think the enum values we would want in that case are the following:
```
NotSubmitted
Submitted
SubmittedLocked
SubmittedAnswersViewable
```
Anyone see something I'm missing? Then maybe call the enum something like `SubmissionState`?

To make things a little more interesting, for lockable stages after being unsubmitted or in the view_answers state for 24 hours, we want to become locked. Right now I do that without an extra db write. In the case of our enum, this would mean we could have a situation where `submission_state=NotSubmitted` and `unlocked_at` is > 24 hours ago, such that we want to behave like `submission_state=SubmittedLocked`